WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingHeaderView.swift

```diff
@@ -22,24 +22,24 @@ class PrepublishingHeaderView: UIView, NibLoadable {
         blogTitleLabel.text = blog.title
     }
 
-    // MARK: - Back button
+    // MARK: - Close button
 
-    func hideBackButton() {
-        backButtonView.layer.opacity = 0
-        backButtonView.isHidden = true
+    func hideCloseButton() {
```

Comment:
Is there a reason for 2 different methods versus just using 1 that takes a `hidden` argument?

Reply:
Done!